.github/workflows/ispc-ci.yml

```diff
@@ -496,6 +549,48 @@ jobs:
         name: fail_db.llvm17.${{matrix.target}}.txt
         path: fail_db.txt
 
+  # Test xe release version
+  linux-test-xe-llvm16-release:
+    needs: [define-flow, linux-build-ispc-xe-llvm16-release]
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    strategy:
+      fail-fast: false
+    env:
+      INSTALL_COMPUTE_RUNTIME: 1
```

Comment:
Since you run only CPU testing you don't need to install Compute Runtime here. But if it is done "to not forget in the future when we might have runner with GPU", I'm OK with it, just leave the comment.

Reply:
If I remember correctly, `ocloc` compiler is needed for `lit-tests`.

Reply:
Yes, so for build tasks which include lit testing, CR is needed, but for testing jobs - it's not needed IMO.

Reply:
Disabled `INSTALL_COMPUTE_RUNTIME` for test-xe.